referenceframe/model.go

```diff
@@ -228,11 +227,22 @@ func (m *SimpleModel) inputsToFrames(inputs []Input, collectAll bool) ([]*static
 		}
 		multierr.AppendInto(&err, errNew)
 		if collectAll {
-			tf, err := NewStaticFrameFromFrame(transform, composedTransformation)
+			var geometry spatialmath.Geometry
+			gf, err := transform.Geometries(input)
 			if err != nil {
 				return nil, err
 			}
-			poses = append(poses, tf.(*staticFrame))
+			geometries := gf.Geometries()
+			if len(geometries) == 0 {
+				geometry = nil
+			} else {
+				geometry = geometries[0]
```

Comment:
Why are we discarding any geometries beyond the first?

Reply:
Until we do the correct thing and not just treat the geometries as the sphere you get from revolving them, I would like to keep this simple

Reply:
We don't support the ability to put multiple geometries in a static frame

Reply:
Ah, I see; it was already doing this discard, this merely formalizes it.

Part of the incoming backend for GetGeometries will require removing the single geometry restriction, could you add a TODO here to remind me to update this when I do that?

Reply:
Yep adding